deploy/nebula-cluster/values.yaml

```diff
@@ -0,0 +1,91 @@
+terminationPolicy: Delete
+nebula:
+  version: v3.5.0
+  imagePullPolicy: Always
+  storageClassName: ""
+  enablePVReclaim: false
+  enableBR: false
+  enableForceUpdate: false
+  schedulerName: default-scheduler # nebula-scheduler
+  unsatisfiableAction: ScheduleAnyway
+  logRotate: { }
+  reference:
+    name: statefulsets.apps
+    version: v1
+  graphd:
+    image: docker.io/vesoft/nebula-graphd:v3.5.0
+    replicas: 1
+    serviceType: NodePort
+    env: []
+    config: {}
+    resources:
+      requests:
+        cpu: "500m"
+        memory: "500Mi"
+      limits:
+        cpu: "1"
+        memory: "500Mi"
+    logStorage: "500Mi"
+    podLabels: {}
+    podAnnotations: {}
+    nodeSelector: {}
+    tolerations: []
+    affinity: {}
+    readinessProbe: {}
+    initContainers: []
+    sidecarContainers: []
+    sidecarVolumes: []
```

Comment:
tidy up required values for this chart.

Reply:
Removed empty variables.

Reply:
expected to preserve values used in the template, also it will be ideal to document the values items, plz reference `deploy/helm/values.yaml` example

Reply:
same for `deploy/nebula/values.yaml`

Reply:
Added some comments, removed all unused variables to avoid confusion. Previously, some values are ported form nebula-operator's configs.